src/dotnet/APIView/APIView/Model/CodeFile.cs

```diff
@@ -151,23 +141,20 @@ public static async Task<CodeFile> DeserializeAsync(Stream stream, bool hasSecti
 
         public async Task SerializeAsync(Stream stream)
         {
-            if (this.APIForest.Count > 0)
-            {
-                using (var tempStream = new MemoryStream())
-                {
-                    await JsonSerializer.SerializeAsync(tempStream, this, _treeStyleParserSerializerOptions);
-                    tempStream.Position = 0;
+            await JsonSerializer.SerializeAsync(stream, this, _serializerOptions);
+        }
 
-                    using (var compressionStream = new GZipStream(stream, CompressionMode.Compress, leaveOpen: true))
-                    {
-                        await tempStream.CopyToAsync(compressionStream);
-                    }
-                }
-            }
-            else
+        /***GetApiText method will generate complete text representation of API surface to help generating the content.
+        * One use case of this function will be to support download request of entire API review surface.
+        */
```

Comment:
nit: can we convert all these comments on the API surface to proper XML doc comments?

```suggestion
        /// <summary>
        /// Generates a complete text representation of API surface to help generating the content.
        /// One use case of this function will be to support download request of entire API review surface.
        /// </summary>
```